packages/lit-html/src/directive-helpers.ts

```diff
@@ -91,24 +60,47 @@ const createMarker = () => document.createComment('');
  * `refPart`.
  *
  * @param containerPart Part within which to add the new NodePart
+ * @param part Part to insert, or undefined to create a new part
  * @param refPart Part before which to add the new NodePart; when omitted the
  *  part added to the end of the `containerPart`
  */
-export const createAndInsertPart = (
+export const insertPart = (
   containerPart: NodePart,
+  part: NodePart | undefined,
```

Comment:
Nit: Even though there are cases where either will be called with `undefined`, I think flipping `part` and `refNode` makes a little more sense API-wise. Explicitly having to pass `undefined` for `refNode` to mean "at end" seems better than `undefined` for `part` to mean "please create a part".

Reply:
This API is a little rough because all arguments have the same type and two are optional. Made `part` last.